### PR TITLE
Fix 324

### PR DESCRIPTION
**Pull request by jeremychoi**
This PR fixes OWASP/wstg#324 .

- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- fixed spelling errors

Thank you for your contribution!

### CODE REVIEW

File: document/4_Web_Application_Security_Testing/4.3_Configuration_and_Deployment_Management_Testing/4.3.9_Test_File_Permission_WSTG-CONFIG-009.md [hunk 1]
```diff
@@ -23,25 +23,26 @@ Under the linux, use ls command to check the file permission. Alternatively, nam
...
-  Sensitive info files (Key, encryption)  | 600(rw------ ) |
+  database files directory                | 700(rwx------)  |
+  Sensitive info files (Key, encryption)  | 600(rw------ )  |
 
 ## Tools
 
 - [Windows AccessEnum](https://technet.microsoft.com/en-us/sysinternals/accessenum)
 - [Windows AccessChk](https://technet.microsoft.com/en-us/sysinternals/accesschk)
 - [Linux namei](https://linux.die.net/man/1/namei)
+
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.3_Configuration_and_Deployment_Management_Testing/4.3.9_Test_File_Permission_WSTG-CONFIG-009.md]: Remove this blank to address the lint failure

**Comment by jeremychoi** [comment]
Thanks. Let me check again. Interestingly the linter I installed per the
guide on my mac gave a warning when the line wasn't added.

File: document/4_Web_Application_Security_Testing/4.3_Configuration_and_Deployment_Management_Testing/4.3.9_Test_File_Permission_WSTG-CONFIG-009.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 ## Summary
 
-Impoper file permission configuration may result in privilledge escalation, information explousure, DLL injection, or unauthorized file access.
+Improper file permission configuration may result in privilege escalation, information exposure, DLL injection, or unauthorized file access.
 
-Therefore, files permission must be properly configured with minium access permission by default. The file permission needs to be configured include -
+Therefore, files permission must be properly configured with minimum access permission by default. The file permission needs to be configured include -
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.3_Configuration_and_Deployment_Management_Testing/4.3.9_Test_File_Permission_WSTG-CONFIG-009.md]: Can you update this sentence? It's pretty much wrong.
Not required from you, we can open an issue if this is something you're not interested in doing.

Comment (jeremychoi) [reply document/4_Web_Application_Security_Testing/4.3_Configuration_and_Deployment_Management_Testing/4.3.9_Test_File_Permission_WSTG-CONFIG-009.md]: Let me try.

Comment (jeremychoi) [reply document/4_Web_Application_Security_Testing/4.3_Configuration_and_Deployment_Management_Testing/4.3.9_Test_File_Permission_WSTG-CONFIG-009.md]: i tried to go a bit further. please have a look at the latest one and let me know whether it looks good or got worse...

**Pull request merged**